packages/account/src/Sections/Verification/ProofOfAddress/proof-of-address.tsx

```diff
@@ -1,19 +1,18 @@
-import DemoMessage from 'Components/demo-message';
-import { PlatformContext } from '@deriv/shared';
-import ProofOfAddressContainer from './proof-of-address-container.jsx';
 import React from 'react';
 import { observer, useStore } from '@deriv/stores';
+import DemoMessage from '../../../Components/demo-message';
+import ProofOfAddressContainer from './proof-of-address-container';
 
 const ProofOfAddress = observer(() => {
     const { client, notifications, common } = useStore();
     const { app_routing_history } = common;
     const { is_virtual, landing_company_shortcode, has_restricted_mt5_account, is_switching } = client;
     const { refreshNotifications } = notifications;
-    const { is_appstore } = React.useContext(PlatformContext);
-    if (is_virtual) return <DemoMessage has_demo_icon={is_appstore} has_button={true} />;
+    if (is_virtual) return <DemoMessage has_button={true} />;
 
     return (
         <ProofOfAddressContainer
+            data-testid='ProofOfAddress'
```

Comment:
data-testid='dt_proof_of_address'

Reply:
Do we use this testid anywhere? I couldnt find a spec file using this? Can you help to check this?

Reply:
no, Amina.  I forget to remove it while refactoring and testing my changes.
thank you